cookbook/case_studies/feature_engineering/feast_integration/README.rst

```diff
@@ -0,0 +1,82 @@
+Feast Integration
+-----------------
+
+**Feature Engineering** off-late has become one of the most prominent topics in Machine Learning.
+It is the process of transforming raw data into features that better represent the underlying problem to the predictive models, resulting in improved model accuracy on unseen data.
+
+**Feast (Feature Store) is an operational data system for managing and serving machine learning features to models in production.**
+
+Flyte provides a way to train models and perform feature engineering as a single pipeline.
+But, it provides no way to serve these features to production when the model matures and is ready to be served in production.
+
+This is where the integration between Flyte and Feast can help users take their models and features from prototyping all the way to production cost-effectively and efficiently. 🚀
+
+In this tutorial, we'll walk through how Feast can be used to store and retrieve features to train and test the model curated using the Flyte pipeline.
+
+Dataset
+=======
+We'll be using the horse colic dataset wherein we'll determine if the lesion of the horse is surgical or not. This is a modified version of the original dataset.
+
+The dataset will have the following columns:
+
+.. list-table:: Horse Colic Features
+    :widths: 25 25 25 25 25
+
+    * - surgery
+      - Age
+      - Hospital Number
+      - rectal temperature
+      - pulse
+    * - respiratory rate
+      - temperature of extremities
+      - peripheral pulse
+      - mucous membranes
+      - capillary refill time
+    * - pain
+      - peristalsis
+      - abdominal distension
+      - nasogastric tube
+      - nasogastric reflux
+    * - nasogastric reflux PH
+      - rectal examination
+      - abdomen
+      - packed cell volume
+      - total protein
+    * - abdominocentesis appearance
+      - abdomcentesis total protein
+      - outcome
+      - surgical lesion
+      - timestamp
+
+The horse colic dataset will be a compressed zip file consisting of the SQLite DB.
+
+Why SQLite3?
+^^^^^^^^^^^^
+SQLite3 is written such that the task doesn't depend on the user's image. It basically:
```

Comment:
i think this detail is irrelevant for this part?
we should just say that we wanted a simple `SQL` like system that can work locally on everyones laptops. But this can as easily be replaced by Snowflake, Athena, Hive, BigQuery or even Spark